assembl/static2/js/app/pages/multiColumnsAdmin.jsx

```diff
@@ -54,7 +53,7 @@ class MultiColumnsAdmin extends React.Component {
             annotation="multicolumnAnnotation"
           />
         )}
-        {!isNaN(currentStep) && <Navbar currentStep={currentStep} totalSteps={1} phaseIdentifier="multiColumn" />}
+        {section && <Navbar currentStep={section} steps={['1']} phaseIdentifier="multiColumn" />}
```

Comment:
The `phaseIdentifier` should now use the one from constants, no?

Reply:
yeah, and should probably be multiColumns with a S